crates/symbolicator/src/endpoints/symbolicate.rs

```diff
@@ -38,6 +38,7 @@ impl ConfigureScope for SymbolicationRequestQueryParams {
 /// JSON body of the symbolication request.
 #[derive(Serialize, Deserialize)]
 pub struct SymbolicationRequestBody {
+    pub platform: Option<Platform>,
```

Comment:
I believe this should also have a `#[serde(default)]`?

Reply:
`#[serde(default)]` is already implied for `Option`.